src/coordination/coordinator_instance.cpp

```diff
@@ -68,6 +72,7 @@ CoordinatorInstance::CoordinatorInstance()
                 repl_instance.StopFrequentCheck();
               }
               auto lock = std::unique_lock{coord_instance_lock_};
+              std::ranges::for_each(repl_instances_, [](auto &repl_instance) { repl_instance.StopFrequentCheck(); });
               repl_instances_.clear();
               spdlog::info("Stopped all replication instance frequent checks.");
```

Comment:
I think we should put trace log-level logs

Reply:
Done